README.md

```diff
@@ -2,6 +2,36 @@
 
  Allows use of plain strings and files as credentials to be passed to miscellaneous build steps. See also this [plugin's wiki page][wiki]
 
+## Description
+
+This plugin provides two new types of credentials:
+
+-   `FileCredentials` or *Secret file*:
+
+<p align="center">
```

Comment:
Use markdown markup for images and I think you need to link with a / at the start of the hyperlink to make it work on the plugin site, I.e /docs not docs